plugin/src/main/resources/io/jenkins/plugins/analysis/core/model/JobAction/floatingBox.jelly

```diff
@@ -1,6 +1,18 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:c="/charts">
+<j:jelly xmlns:j="jelly:core" xmlns:c="/charts" xmlns:i="/issues">
 
-  <c:trend-chart it="${from}" title="${from.trendName}" enableLinks="true"/>
+ <j:choose>
```

Comment:
Seems that this change breaks 2 existing tests: https://github.com/jenkinsci/warnings-ng-plugin/pull/436/checks?check_run_id=571774271

Reply:
The tests are expecting a chart which is not there anymore.